taipy/__init__.py

```diff
@@ -13,22 +13,22 @@
 
 if find_spec("taipy"):
     if find_spec("taipy.config"):
-        from taipy.config._init import *  # type: ignore
+        from taipy.config._init import *  # noqa: F403
```

Comment:
❓ Is it redundant with the global config here ? https://github.com/Avaiga/taipy/pull/626/files#diff-26f436cb29eecc7bb4f9066747b43df8da81ac808955ce95c109f5d7aa778989R48

Reply:
True. I added this before the global rule. This should be removed